public/stylesheets/app/_case-progress-card.scss

```diff
@@ -0,0 +1,99 @@
+.app-summary-card {
```

Comment:
Lift and shift from https://github.com/ministryofjustice/hmpps-manage-supervisions/blob/c08997614f346c9f331dc94cdbdb9fdcc3de25c0/src/client/sass/components/_summary-card.scss